src/Serialize.h

```diff
@@ -25,6 +31,15 @@ namespace drafter {
         UnknownASTType = -1
     };
 
+#ifdef _WITH_REFRACT_
+    /**
+     * Hold all **Named Types** converted to Refract Element
+     * Later use this registry for expanding element and rendering before serialization
+     */
+
+    static refract::Registry NamedTypesRegistry;
```

Comment:
This is not good, 
- do not use global variables
- `static`  in header is typical reason of "duplicit symbols" while linking.

Do we really need use it in more files?